pkg/nimble/Makefile.dep

```diff
@@ -122,6 +122,12 @@ ifneq (,$(filter nimble_netif_ext,$(USEMODULE)))
   USEMODULE += nimble_adv_ext
 endif
 
+ifneq (,$(filter nimble_netif,$(USEMODULE)))
+  ifneq (,$(filter nimble_autoadv,$(USEMODULE)))
+    USEMODULE += nimble_adv_ext
```

Comment:
I think a small comment on why this is a necessary dependency should be added.

Reply:
Added comment